src/features/NewOrder/NewOrder.js

```diff
@@ -0,0 +1,300 @@
+//WORK ITEMS
+
+// Drop down suggestions for stock symbol input.
+// keep input value (watched) in state?
+// show display if match
+
+// imports
+import React, { useState, useRef } from 'react';
+import { useForm } from 'react-hook-form';
+
+// Styles
+import styles from './NewOrder.module.css';
+
+// sample data
+const realMemeStocks = ['GMSK', 'HYPE', 'MEME', 'MDPC', 'OTHR'];
+const ownedStocks = [
+  {
+    stockName: 'GMSK',
+    stockShares: 7,
+  },
+  {
+    stockName: 'HYPE',
+    stockShares: 23,
+  },
+  {
+    stockName: 'MEME',
+    stockShares: 42,
+  },
+];
+
+const totalCash = 5000.5;
+
+export default function NewOrder() {
+  // use state for showing buy or sell info
+
+  const [showBuyForm, setShowBuyForm] = useState('true');
```

Comment:
This should be a boolean value not a string representation of a boolean. The way you use `setShowBuyForm` below changes the values to be booleans so it is a good idea to keep your types consistent.

Also in these cases it is often a good idea to have a toggle function (or even a custom hook) like so:
`const toggleShowBuyForm = () => setShowBuyForm((prev) => !prev);`